Common/Interfaces/IApi.cs

```diff
@@ -132,7 +132,32 @@ public interface IApi : IDisposable
         /// </summary>
         /// <returns>List of live algorithm instances</returns>
         LiveList LiveList();
-        
+
+        /// <summary>
+        /// Gets the logs of a specific live algorithm 
+        /// </summary>
+        /// <param name="projectId">Project Id of the live running algorithm</param>
+        /// <param name="algorithmId">Algorithm Id of the live running algorithm</param>
+        /// <returns>List of strings that represent the logs of the algorithm</returns>
+        LiveLog ReadLiveLogs(int projectId, string algorithmId);
```

Comment:
Please pass in a start, end and format specifier:

format - raw or json
start (api takes timestamp start)
end (api takes timestamp end)

The C# method should have a datetime input which is converted to -> timestamp in unix sec. Defaulting the start to 0 and end to unix-now.